### ISSUE TITLE
Many ESLint  errors of "...is defined but never used" for JSX tags, after update to 6.0.0

### ISSUE THREAD

**Issue by metamas** [mozilla-neutrino/neutrino-dev#273]
I recently updated from neutrino `5.9.0` to `6.0.0` and suddenly everywhere I have imported a component and called as a JSX tag is throwing the ESLint error `...is defined but never used      no-unused-vars`.

My preset packages have also been updated to `6.0.0` and my `.neutrinorc.js` is

```js
module.exports = {
  use: ['neutrino-preset-airbnb-base', 'neutrino-preset-react']
};
```

My project is structured with a s`src/index.js` and all of my components live in `src/js` and have a `.js` file extension. An example of one of my component files would be

```jsx
import React from 'react';
import Lower from './Lower';
import preload from '../data.json';

const stuff = preload.stuff;

class Higher extends React.Component {
  render() {
    return (
      <div className='higher'>
        {stuff
          .filter(thing => thing.foo === this.props.bar)
          .map(thing => <Lower {...thing} />)}
      </div>
    );
  }
}

export default Higher;
```
Which upon running `npm start` will yield the error output below

```
ERROR in ./src/js/Higher.js

/path/to/my/project/src/js/Higher.js
  2:8  error  'Lower' is defined but never used      no-unused-vars

✖ 1 problems (1 errors, 0 warnings)

 @ ./src/js/App.js
 @ ./src/index.js
 @ multi ./~/react-hot-loader/patch.js (webpack)-dev-server/client?http://localhost:5000 (webpack)/hot/dev-server.js ./src/index
```

This wasn't happening until after the update. I am not yet convinced it's a bug, but instead might be something I am doing wrong. Any insight/suggestions is much appreciated.

**Comment by eliperelman** [comment]
@metamas hello! This actually all caused by an oversight in the React preset in Neutrino v5.x. With v5, we force-included `react:recommended` when using ESLint with the React preset, without thinking about the effects of this. As such, it was removed in v6.

Now to the issue. The **airbnb-base** preset doesn't have any knowledge of React or JSX, so it throws these errors. If you use neutrino-preset-airbnb, you should see these go away.

Give that a shot and let me know if that resolves it.

**Comment by metamas** [comment]
@eliperelman Is that to say I should use `neutrino-preset-airbnb` in place of `neutrino-preset-airbnb-base`, or _in additon to_?

I expected that my use of `neutrino-preset-react` along with, and following, `neutrino-preset-airbnb-base` in the `use` array of my `.neutrinorc.js` would take care of linting for React stuff. Is that the wrong way to think about it? The `neutrino-preset-airbnb-base` docs even site "Modern Babel knowledge supporting ES modules, **JSX (when used with React preset)**, Web and Node.js apps" as a feature.

**Comment by eliperelman** [comment]
They are exclusive presets, so you can just remove the airbnb-base and switch to airbnb. Fixing in #275.

**Comment by eliperelman** [comment]
Should be fixed in #275. Released in v6.0.4.

**Comment by xiongchengqing** [comment]
I think this should reflect to the official docs [Neutrino Airbnb Preset](https://neutrino.js.org/presets/neutrino-preset-airbnb-base/). People using neutrino want to save their times, but this out of expectation would cost more time to google and solve it.
